internal/metrics/metrics_gitsync.go

```diff
@@ -0,0 +1,48 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	GitSyncFailed = promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "ocp_git_sync_failed_total",
```

Comment:
What's the benefit of having a "total" suffix? Or is this some convention I'm not aware of? I had been reading the kprom sources today and [they specifically dropped it.](https://github.com/twmb/franz-go/pull/1071). However, the [prom docs](https://prometheus.io/docs/practices/naming/) suggest that it's a good name, I think? Let's keep it.

Alternatively, couldn't the state be a label? We have `ocp_git_sync_count_total` below, perhaps `state = "success" | "error"` could be useful... What do you think?